README.md

```diff
@@ -215,15 +226,15 @@ project specific naming convention for this e.g. `remote-<your name>`
 The name of the git remote for the GitHub project which has the Continuous Pipe integration.
 In most cases you will have cloned the project repo from this so this will be `origin`.   
  
-### What is the default container for the watch, bash and resync commands? (DEFAULT_CONTAINER)     
+### What is the default container for the watch, bash, fetch and resync commands? (DEFAULT_CONTAINER)     
  
-This is an optional setting, if provided this will be used by the `bash`, `watch` and `resync` commands as
-the container you connect to, watch for file changes or resync with respectively unless you provide
+This is an optional setting, if provided this will be used by the `bash`, `watch`, `fetch` and `resync` commands as
+the container you ssh onto, watch for file changes or resync with respectively unless you provide
```

Comment:
@kierenevans I've updated it to say connect to, which was the change already made that was lost in a merge I think.

@tonypiper because of the git meanings? Maybe it should be something else altogether ion that case. This was the name suggested by the users but can see why it may be an issue.